src/main/java/com/example/springpractise/service/SensitiveWordService.java

```diff
@@ -0,0 +1,55 @@
+package com.example.springpractise.service;
+
+import com.example.springpractise.model.SensitiveWord;
+import com.example.springpractise.repository.SensitiveWordRepository;
+import org.springframework.stereotype.Service;
+
+import javax.persistence.EntityNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+@Service
+public class SensitiveWordService {
+    private final SensitiveWordRepository sensitiveWordRepository;
+    public SensitiveWordService(SensitiveWordRepository sensitiveWordRepository){
+        this.sensitiveWordRepository=sensitiveWordRepository;
+    }
+
+    public SensitiveWord  createSensitiveWord(String newWord){
+        try {
+           return sensitiveWordRepository.save(new SensitiveWord(newWord));
+        }catch (Exception E){
+            throw new IllegalArgumentException();
+        }
+    }
+
+    public List<SensitiveWord> getSensitiveWords() {
+        Iterable<SensitiveWord> sensitiveWordIterable = sensitiveWordRepository.findAll();
+        List<SensitiveWord> result = new ArrayList<>();
+        sensitiveWordIterable.forEach(result::add);
+
+        if(result.size()>0){
+            return result;
+        }
+        else{
```

Comment:
Returning an empty list is a valid response so you can just return the empty list instead of throwing an exception